bitnami/kong/Chart.yaml

```
@@ -34,4 +34,4 @@ name: kong
 sources:
   - https://github.com/bitnami/bitnami-docker-kong
   - https://konghq.com/
-version: 3.4.8
+version: 3.4.9
```

Since it is a feature addition instead of a bug fixing, can you bump the minor version?
```suggestion
version: 3.5.0
```